### ISSUE TITLE
Generate current-full.csv and current-federal.csv daily and publish at a public URL

### ISSUE THREAD

**Issue by h-m-f-t** [cisagov/manage.get.gov#1075]
### Issue Description

#996 prepared 3 different reports and made them available as on-demand CSV exports. For this issue, we should provide the capability to generate 2 of those reports – current-full.csv and current-federal.csv – and have them accessible through an API.

### Acceptance criteria

- [ ] Save the files (delete/overwrite any prior copies) and make them available at a public URL
- [ ] Cameron can use the public URL to replace the `http_url` value in our [Git scraping instance](https://github.com/cisagov/dotgov-data/blob/main/.github/workflows/flat.yml).

### Issue Links

⛔ Blocked by #996
Dependency of #1403, #1317 (That ticket is for notification for when this process fails to complete properly.)

**Comment by vickyszuchin** [comment]
Comment for consideration (from Neil during refinement meeting),

Automation in Github Actions is pretty accessible with what we already do.

**Comment by vickyszuchin** [comment]
Decision in refinement meeting (10/12) to break the notification AC into a separate ticket.

cc: @PaulKuykendall
